packages/templates/react-urql-graphql-codegen/src/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/App';
+import {
+  createClient,
+  Provider,
+  debugExchange,
+  cacheExchange,
+  fetchExchange
+} from 'urql';
+
+
+const client = createClient({
```

Comment:
Please use some constant for the endpoint that can be replaced at build time in standard way.